src/FastRouteRouter.php

```diff
@@ -44,6 +45,19 @@
  *     cache_file?: string,
  *     ...
  * }
+ * @psalm-type FastRouteNotFoundResult = array{
+ *     0: Dispatcher::NOT_FOUND,
```

Comment:
I would feel more comfortable with at least constants instead of hard coded index values. But that would probably be a larger scope than this PR intends.

Reply:
These are defined in the FastRoute project itself so it would mean sending PRs there.